group_map.opam

```diff
@@ -0,0 +1,6 @@
+opam-version: "1.2"
```

Comment:
This needs to be a proper opam file, otherwise pinning from this repo will break